src/pages/index.tsx

```diff
@@ -66,6 +66,16 @@ const Home: NextPage = () => {
           >
             Tweet
           </Button>
+
+          <Button
+            colorScheme="gray"
+            mt="10px"
+            as={'a'}
+            href="https://twitter.com/sseraphini?ref_src=twsrc%5Etfw" 
+            target="_blank"
+          >
+            Follow @sseraphini
```

Comment:
just `Follow`

Reply:
also add the twitter icon ?

Reply:
sure, I removed because the other button already has a Twitter icon, but I can add again tho